stellar-node/pvc.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: pv0001
+spec:
+  accessModes:
+    - ReadWriteOnce
+  capacity:
+    storage: 5Gi
```

Comment:
Will 5Gi be enough?

Reply:
Shouldn't this size be a parameter taken from `values.yaml`?

Reply:
I used that file for testing and forgot to remove it later.  You can skip using a pvc.yml unless you are testing volumes.

Reply:
I see that `pv0001` is used in the deployment. Also if PVC is not used isn't all blockchain data going to be lost if the pods are recreated?

Reply:
pvc001.yaml was a test I did for pv001 and is not required for normal operation.  What you do need to do is to configure persistence for the data volume in values.yaml.  The other volume(pv001) is for /var/run which is not used for data, only for pid files and other temporary database server files.  If you lose that volume, you will not lose any data.